tests/integration/multi_table/test_hma.py

```diff
@@ -1370,16 +1370,7 @@ def test_null_foreign_keys(self):
         metadata.validate_data(data)
 
         # Run and Assert
-        err_msg = re.escape(
-            'The data contains null values in foreign key columns. This feature is currently '
-            'unsupported. Please remove null values to fit the synthesizer.\n'
-            '\n'
-            'Affected columns:\n'
-            "Table 'child_table1', column(s) ['fk']\n"
-            "Table 'child_table2', column(s) ['fk1', 'fk2']\n"
-        )
-        with pytest.raises(SynthesizerInputError, match=err_msg):
-            synthesizer.fit(data)
+        synthesizer.fit(data)
```

Comment:
When HMA is updated to support null foreign keys we should add more assertions here